src/pseudopeople/noise_entities.py

```diff
@@ -30,10 +30,11 @@ class __NoiseTypes(NamedTuple):
         noise_functions.choose_wrong_options,
         noise_level_scaling_function=noise_scaling.scale_choose_wrong_option,
     )
-    # copy_from_household_member: ColumnNoiseType = ColumnNoiseType(
-    #     "copy_from_household_member",
-    #     noise_functions.copy_from_household_members,
-    # )
+    copy_from_household_member: ColumnNoiseType = ColumnNoiseType(
+        "copy_from_household_member",
+        noise_functions.copy_from_household_member,
+        additional_column_getter=column_getters.copy_from_household_member_column_getter,
```

Comment:
I don't understand how this is working - don't you need to add this to the `class ColumnNoiseType` attributes alongside additional_parameters, etc?